File: features/bobiz_responses.py
import json
import random

# Load JSON once
with open("./features/bobiz.json", "r", encoding="utf-8") as f:
    data = json.load(f)


async def handle_bobiz(ctx):

    # Example: check if the user is allowed
    if ctx.author.bot:
        return

    # Pick a random sentence
    if ctx.author.id == 733414175556239525:
        chosen_sentence = random.choice(data["hamza"])

    elif ctx.author.id == 1433214878658859163:
        chosen_sentence = random.choice(data["jawhara"])

    else:
        chosen_sentence = random.choice(data["words"])

    # You can add more custom logic here, e.g., user-specific responses
    await ctx.send(f"{ctx.author.mention}, {chosen_sentence}")

File: features/bobiz_responses.js
const fs = require("fs");
// Node.js file system module for reading files

// Load the JSON file containing predefined messages
const data = JSON.parse(fs.readFileSync("./features/bobiz.json", "utf8"));

// Function to handle the "bobiz" command
async function handleBobiz(message) {
    // Ignore messages sent by bots
    if (message.author.bot) return;

    // Get the ID of the message author
    const userId = message.author.id;

    // Get the sentences for this user, or default to the "words" entry
    const sentences = data[userId] || data["words"];

    // Pick a random sentence from the list
    const chosen = sentences[Math.floor(Math.random() * sentences.length)];

    // Reply to the user with their chosen sentence
    await message.reply(`${message.author}, ${chosen}`);
}

// Export the function so it can be used in other files
module.exports = { handleBobiz };
